sdk/src/main/java/io/dapr/client/DaprClientImpl.java

```diff
@@ -1336,6 +1341,32 @@ public Mono<Void> scheduleJob(ScheduleJobRequest scheduleJobRequest) {
         scheduleJobRequestBuilder.setDueTime(iso8601Formatter.format(scheduleJobRequest.getDueTime()));
       }
 
```

Comment:
Could we move all the logic related to `FailurePolicy` creation to a helper method, otherwise the `scheduleJob` method gets really busy.